.github/workflows/tests.yml

```diff
@@ -20,17 +20,22 @@ jobs:
           python-version: ${{ matrix.python }}
       - name: Install
         run: |
-          pipx install poetry
-          poetry env use ${{ matrix.python }}
```

Comment:
Hmm, we need to make sure that the right python version is used here, otherwise this test matrix will be worthless.

Something like this should do:
```
uv venv --python 3.11.6
```